yast/src/modules/SMTData.rb

```diff
@@ -64,8 +63,6 @@ def main
       @SCCcredentials_file = "/etc/zypp/credentials.d/SCCcredentials"
       @NCCcredentials_file = "/etc/zypp/credentials.d/NCCcredentials"
```

Comment:
I think so. This should only happen after upgrade from SLES11 and if the file exists, it means that NCC knew about the system and also SCC knows about it.
Maybe it is really not necessary, but will find out during further testing...